packages/jest-runtime/src/transform.js

```diff
@@ -64,7 +66,9 @@ const getCacheKey = (
     }));
   }
   const configStr = configToJsonMap.get(config) || '';
-  if (typeof preprocessor.getCacheKey === 'function') {
+  const preprocessor = getPreprocessor(config);
```

Comment:
I hate that all of this is called `preprocessor`, it's such a silly name. We should rename the variable to `transformer`.